### PR TITLE
fixed resumes lack in filtering

### PR DESCRIPTION
**Pull request by ikzntsv**
I hope this is final pull request regarding this issue

### CODE REVIEW

File: app/Http/Controllers/MainController.php [hunk 1]
```diff
@@ -69,7 +69,7 @@ public function showVacancies(Request $request)
             ));
         }
 
-        $vacancies = Vacancy::unblockVacancies()->isActive()->paginate();
+        $vacancies = Vacancy::unblockVacancies()->allVacancies()->isActive()->paginate();
```
Comment (Quicks) [inline app/Http/Controllers/MainController.php]: It's correct?

Comment (ikzntsv) [reply app/Http/Controllers/MainController.php]: works as correct in local version

Comment (ikzntsv) [reply app/Http/Controllers/MainController.php]: a lot of options such as block vacancy and so on were added thats why I anticipate handful of bugs regarding new scopes soon

Comment (ikzntsv) [reply app/Http/Controllers/MainController.php]: There is the same scope added to resume and company

**Pull request merged**